Source/WebKit/UIProcess/API/Cocoa/WKUIDelegate.h

```diff
@@ -254,6 +254,22 @@ typedef NS_ENUM(NSInteger, WKDialogResult) {
  */
 - (void)webView:(WKWebView *)webView showLockdownModeFirstUseMessage:(NSString *)message completionHandler:(void (^)(WKDialogResult))completionHandler WK_API_AVAILABLE(ios(13.0));
 
+/**
+ * @abstract Called when the web view is about to present its edit menu.
+ *
+ * @param textField    The web view displaying the menu.
+ * @param animator      Appearance animator. Add animations to this object to run them alongside the appearance transition.
```

Comment:
Spacing looks a bit off.

Reply:
Good catch, fixed! (And also changed the `textField` above to `webView` — it's a really good thing you found this!)